OC Share Sheet/Managers.swift

```diff
@@ -30,15 +30,36 @@ import UIKit
     //MARK: OCCommunication
     @objc class var sharedOCCommunication: OCCommunication {
         struct Static {
-            static let sharedOCCommunication: OCCommunication = OCCommunication()
-        }
-        
-        return Static.sharedOCCommunication
-    }
-}
 
+			static func sharedOCCommunication() -> OCCommunication {
```

Comment:
This way of creating singletons is quite old and common back in the Swift 1.0 days. Since then, Swift now supports static class variables so you might want to change it to:
```Swift
@objc static var shared: OCCommunication = {
// Code
}()
```

or even to respect the responsibilities, you might want to change the way this is being done and create these singletons inside an `extension` like:
```Swift
extension OCCommunication {
    @objc static var shared: OCCommunication = {
        //Code
    }()
}
```